CHANGELOG.md

```diff
@@ -427,8 +427,8 @@ more discussion).
   omitting type piracy, ([#835][github-835]) implemented `isequal` and `hash` for `BoundaryIndex` datatypes.
 
 - **VTK export**: Ferrite no longer extends methods from `WriteVTK.jl`, instead the new types
-  `VTKFile` and `VTKFileCollection` should be used instead. New methods exists for writing to
-  a `VTKFile`, e.g. `write_solution`, `write_cell_data`, `write_node_data`, and `write_projection`.
+  `VTKGridFile` and `VTKFileCollection` should be used instead. New methods exists for writing to
```

Comment:
```suggestion
  `VTKGridFile` and `VTKCollectionFile` should be used instead. New methods exists for writing to
```
for symmetry?

Reply:
See suggestion to remove completely here: https://github.com/Ferrite-FEM/Ferrite.jl/pull/1015